src/client/components/layouts/GamePage/Ongoing.tsx

```diff
@@ -1,5 +1,113 @@
-import './GamePage.css'
+import './Ongoing.css'
+import { useState, useEffect, useContext } from 'react'
+import Button from '../../elements/Button/Button'
+import { getGameCards, validateAnswer } from '../../../../api/card'
+import { GameContext, GameResultItem } from '../../../../api/GameContext'
+import { useNavigate } from 'react-router-dom'
+import { FlashcardProps } from '../../elements/Flashcard/Flashcard'
+
+function mapCardToGameResultItem(cards: FlashcardProps[]): GameResultItem[] {
```

Comment:
`FlashcardProps` ist derzeit ein Durchlauferhitzer, oder nicht? 
Würde GameResultItem bzw. FlashcardProps ausreichen?
Über die API wird von CardProps auf FlashcardProps gemapped, um die `back` propertie zu verbergen. Kurz darauf hier auf GameResultItem gemapped aber nie in der Form von FlashcardProps verwendet. 

Eventuell übersehe ich etwas, jedoch habe ich das gefühl, wir könnten es mit einem Type auch lösen?

Reply:
Da hast du recht. Tobi hat ja auch gemeint, dass wir die cards bei der Validierung eigentlich gar nicht fetchen müssen. Das ist ein issue, welches wir noch beheben müssten, um einen besseren flow im prozess zu haben

Reply:
Issue #25